util/export_gkube_images.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# export discovery image
+discovery_pod="$(gkube kubectl get pods | grep discovery  | head -n 1 | awk '{print $1}')"
+discovery_image=$(gkube kubectl describe pod "$discovery_pod"  | grep docker.greenkeytech.com/discovery | awk '{print $NF}' | head -n 1)
+docker exec default-gkube-server ctr images export - "$discovery_image" | docker load
+
+# export init_discovery image
+init_discovery_image=$(gkube kubectl describe pod "$discovery_pod"  | grep docker.greenkeytech.com/greenkey-discovery-sdk-private | awk '{print $NF}' | head -n 1)
```

Comment:
like this @pmahoney ?
```suggestion
init_discovery_image=$(gkube kubectl get deploy gk-discovery -o go-template='{{range .spec.template.spec.initContainers}}{{if eq "initdiscovery" .name}}{{.image}}{{end}}{{end}}')
```